templates/repo/diff/conversation.tmpl

```diff
@@ -1,21 +1,41 @@
 {{$resolved := (index .comments 0).IsResolved}}
+{{$invalid := (index .comments 0).Invalidated}}
 {{$resolveDoer := (index .comments 0).ResolveDoer}}
 {{$isNotPending := (not (eq (index .comments 0).Review.Type 0))}}
+{{$referenceUrl := printf "%s#%s" $.Issue.Link (index .comments 0).HashTag}}
 <div class="conversation-holder" data-path="{{(index .comments 0).TreePath}}" data-side="{{if lt (index .comments 0).Line 0}}left{{else}}right{{end}}" data-idx="{{(index .comments 0).UnsignedLine}}">
-	{{if $resolved}}
+	{{if or $invalid $resolved}}
 		<div class="ui attached header resolved-placeholder gt-df gt-ac gt-sb">
 			<div class="ui grey text">
```

Comment:
```suggestion
			<div class="ui grey text gt-df gt-ac gt-gap-2">
```

Should properly vertically center the icons. Please test if `gt-gap-2` is actually necessary to have some spacing in between inline nodes.